core/storage/suggestion/gae_models.py

```diff
@@ -168,7 +168,7 @@ class GeneralSuggestionModel(base_models.BaseModel):
     final_reviewer_id = datastore_services.StringProperty(indexed=True)
     # The change command linked to the suggestion. Contains the details of the
     # change.
-    change_cmd = datastore_services.JsonProperty(required=True)
+    change = datastore_services.JsonProperty(required=True)
```

Comment:
You shouldn't change this. It will break all existing GeneralSuggestionModels in the production datastore. Perhaps leave as-is and add a comment saying that this maps to `change` in the domain object (and its old name is kept here to preserve backward-compatibility with existing datastore models).

Reply:
Sure, I'll revert this :)